website/content/docs/integrations/vault/index.mdx

```diff
@@ -0,0 +1,60 @@
+
+# Credentials
+Vault can work with Boundary to be a credential [store](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/credential-stores) and [library](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/credential-libraries), which allows for credentials to be stored in Vault and used by Boundary. There are two options here for configuration:
+
+- Generic Secrets.
+- SSH Certificates.
+
+The Generic Secrets references a Vault Key-Value path, where static secrets are stored and can be used to broker credentials to the end-user when connecting to a target.
+
+The SSH Certificates takes advantage of Vault being the CA for SSH certificates by utilising the [HashiCorp Vault SSH Secrets Engine](https://developer.hashicorp.com/vault/docs/secrets/ssh) and can inject SSH certificates into the Boundary session.
+
+It is worth detailing that users must bring their own Vault deployment to Boundary.
+
+## Brokered credentials
+
+This feature was added in Boundary 0.4 and facilitates Boundary as a credential broker for infrastructure targets, by binding credentials with user sessions, and surfacing those credentials during session initialization, with the help of Vault.
+
+An example workflow and use case for this feature would be database access. Certain teams within an organisation, such as finance, need to access sensitive data residing in a database. Irrespective of the frequency that the database needs to be accessed, having long-lived credentials that could be purposely or accidentally leaked, can pose a huge risk to a company.
+
+If Boundary and Vault is added into this workflow, we would be mitigating against this potentially security risk. The Vault database secrets engine can be enabled, with the relevant database plugin configured, to then be able to issue dynamic and short-lived database credentials to the end-user. Organisations can set a limit on the amount of time these credentials are valid and once they have timed-out, access is revoked.
+
+![Brokered Credentials](/img/brokered-creds-ui.png)
+
+Upon connecting to the database through Boundary, Boundary displays the newly generated credentials and displays to the end user to connect to the target.
+
+## Injected credentials
+
+Currently SSH certificate injection is the only type of injection possible with the integration between Boundary and Vault. 
+
+Vault is configured to act as the certificate authority (CA), to ensure that the users’ authenticity is determined by a certificate signed by a trusted CA. When key pairs are generated dynamically, they are signed by Vault to then be used to access the resources. There are two options for the key pair generation, depending on the type of Vault endpoint that is used within the Boundary credential library for SSH certification using Vault.
```

Comment:
```suggestion
Vault is configured to act as the certificate authority (CA), to ensure that the users’ authenticity is determined by a certificate signed by a trusted CA. When key pairs are generated dynamically, they are signed by Vault to then be used to access the resources. There are two options for the key pair generation, depending on the type of Vault endpoint that you use within the Boundary credential library for SSH certification.
```

Reply:
I think the *using Vault* part at the end of the last sentence is implied, but please check me to make sure I didn't change the meaning here.

Reply:
Agreed! Your edit still makes sense :)